client/rpc/src/state/mod.rs

```diff
@@ -94,6 +94,13 @@ where
 		key: StorageKey,
 	) -> FutureResult<Option<StorageData>>;
 
+	/// Returns storage entries for multiple keys at a specific block's state.
+	fn storage_entries(
+		&self,
+		block: Option<Block::Hash>,
+		keys: Vec<StorageKey>,
+	) -> FutureResult<Vec<Option<StorageData>>>;
+
```

Comment:
For what do we need this function?

Reply:
bc `StateApi` impl for `State` requires it.

Reply:
Can you please link me to what this requires?

Reply:
https://github.com/paritytech/substrate/blob/267dac2afdef51554eeb881fac0d844dd2761c92/client/rpc/src/state/mod.rs#L266

Reply:
And where does it need this function in the implementation there?

Reply:
```rust
error[E0046]: not all trait items implemented, missing: `storage_keys`
   --> client/rpc/src/state/mod.rs:273:1
    |
273 | / impl<Block, Client> StateApi<Block::Hash> for State<Block, Client>
274 | | where
275 | |     Block: BlockT + 'static,
276 | |     Client: Send + Sync + 'static,
...   |
431 | |     }
432 | | }
    | |_^ missing `storage_keys` in implementation
    |
    = help: implement the missing item: `fn storage_keys(&self, _: StorageKey, _: std::option::Option<Hash>) -> Pin<Box<(dyn futures::Future<Output = Result<Vec<StorageKey>, sc_rpc_api::state::error::Error>> + std::marker::Send + 'static)>> { todo!() }`
    ```

Reply:
Sorry, it tracked the wrong function, forget the above comments.

Reply:
The function is not required for the RPC call. 
I implemented them to have the same set of query possibilities, if you say we don't need them I can remove them.

Reply:
Yeah please remove it.